<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMTDControllerForm</class>
 <widget class="QWidget" name="VMTDControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="_1">
     <item>
      <widget class="QPushButton" name="pbSettings">
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStartController">
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStopController">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="_3">
     <item>
      <layout class="QHBoxLayout" name="_4">
       <item>
        <spacer name="_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbRefresh">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="_11">
       <property name="text">
        <string>Сетевые устройства с поддержкой NX-API:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwNxApiAdapters"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_6">
       <item>
        <widget class="QLabel" name="_7">
         <property name="text">
          <string>Управление узлами:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNodeServer">
         <property name="text">
          <string>Сервер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNodeClient">
         <property name="text">
          <string>Клиент</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
